content/articles/visualizing-repositories-using-pygal/index.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /visualizing-repositories-using-pygal/
+title: Visualizing repositories using Pygal
+description: In this article, you'll learn how to construct a visualization that shows the comparative popularity of Python works on GitHub using Pygal.
+author: duncan-ndegwa
+date: 2021-09-03T00:00:00-04:35
+topics: [API]
+excerpt_separator: <!--more-->
+images:
+
+ - url: engineering-education\content\articles\visualizing-repositories-using-pygal\hero.jpg
+   alt: Visualizing repositories using Pygal hero image
+---
+
+### Introduction
+Pygal allows us to create a range of graphs and charts. In this article, you'll learn how to construct a visualization that shows the comparative Python's prominence works on github.com. We'll create an interactive bar chart, where the height of each bar will represent the sum of stars earned by the project. By clicking a bar, you'll be sent to the project's GitHub page.
+<!--more-->
+
+### Prerequisites
+You'll need some Python skills to follow along with this tutorial. You must be in a position to work with web APIs in Python. Also, you need to have [PIP](https://pip.pypa.io/en/stable/installation/) installed.
```

Comment:
```suggestion
You'll need some Python skills to follow along with this tutorial. You must be in a position to work with web APIs in Python. Also, you need to have [PIP](https://pip.pypa.io/en/stable/installation/) installed.
```
You can break the prerequisites as a list items:
Some knowledge of Python
Knowledge on Web APIs
Pip installed